data-plane/benchmarks/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/UnorderedOffsetManagerBenchmark.java

```diff
@@ -71,96 +73,70 @@ public void doSetup() {
 
   @Benchmark
   public void benchmarkReverseOrder(RecordsState recordsState, Blackhole blackhole) {
-    UnorderedOffsetManager offsetManager = new UnorderedOffsetManager(new MockKafkaConsumer(), null);
+    OffsetManager offsetManager = new OffsetManager(Vertx.vertx(), new MockKafkaConsumer(), null, 10000L);
 
     int partitions = 100;
     for (int partition = 0; partition < partitions; partition++) {
-      blackhole.consume(
-        offsetManager.recordReceived(recordsState.records[partition][0])
-      );
+      offsetManager.recordReceived(recordsState.records[partition][0]);
```

Comment:
are they directly consumed?

Reply:
I updated the code to make it compile but I'm inclined to drop these `OffsetManager` benchmarks since they are not really useful to catch perf regressions as they are.